sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -27,7 +27,12 @@ export function isTokenCredential(credential: any): credential is TokenCredentia
 
 // @public
 export interface TokenCredential {
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    getToken(requestContext: TokenRequestContext, options?: GetTokenOptions): Promise<AccessToken | null>;
+}
+
+// @public
+export interface TokenRequestContext {
+    scopes: string | string[];
```

Comment:
From issue #5625,

> getToken should use TokenRequestContext interface as first parameter to take scopes and parent request ID

@bterlson, @daviwil, @xirzec Does this mean that `spanOptions` moves from `GetTokenOptions` to `TokenRequestContext`?

Reply:
Confirmed with @schaabs that the parent request id is not related to span options.
And that the parent request id can be an additive change post the upcoming release